launch/model.py

```diff
@@ -0,0 +1,19 @@
+from typing import Dict
+
+from pydantic import BaseModel, Field
+
+
+class ModelDownloadRequest(BaseModel):
```

Comment:
possible to reduce # of custom classes if possible? i'm thinking we should make this client as thin as possible so in the future we could solely depend on autogen

Reply:
I'm not sure I understand completely. Do you mean I should repurpose one of the other existing Request/Response classes for ModelDownload to avoid having new ones written? I think the way I have it currently is just templated based on how we set up other requests / responses. I'm happy to change it, I just want to make sure I understand how it needs to be changed.